examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -12,9 +12,8 @@
    "metadata": {},
```

Comment:
<p>Rename to inference_pipeline</p><p>Remove targets from predict_model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>